x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -108,7 +108,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    * Enables expandable flyout in timeline
    */
-  expandableTimelineFlyoutEnabled: false,
+  expandableTimelineFlyoutEnabled: true,
```

Comment:
this should not be changed, please revert. The expandable flyout isn't ready to be turned on by default on `8.14`

Reply:
Thanks @PhilippeOberti! I'm not sure why this was enabled, but we will verify to ensure it wasn't a requirement for getting the new assistant flyout to work within Timeline. 👍

Reply:
Resolved in [2b237db](https://github.com/elastic/kibana/pull/176657/commits/2b237dbc4edc2ed6cc96477dc6610adf13cfb578) 👍